src/main/java/TaskList.java

```diff
@@ -0,0 +1,78 @@
+import java.util.ArrayList;
+
+public class TaskList {
+
+    public ArrayList<Task> tasks = new ArrayList<>();
+
+    Ui ui = new Ui();
```

Comment:
I think it doesn't really make sense to _have_ a Ui in your TaskList. You could consider sending your ui outputs out from this class first to a higher level that sends them to the Ui, or you could make your Ui methods static such that you don't have to explicitly create a Ui object in every file you want to use it.